<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaFenetre</class>
 <widget class="QWidget" name="MaFenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaFenetre</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="readerName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="5">
    <widget class="QLabel" name="logo">
     <property name="text">
      <string>logo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Programme lecture/ecriture carte Mifare</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="connect">
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="quit">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <widget class="QLineEdit" name="userFirstNameEditor"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="format">
       <property name="text">
        <string>Formater la carte</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>Nom    </string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="countTripEditor"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="userNameEditor"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="countTripLabel">
       <property name="text">
        <string>Nombre voyages       </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="userFirstNameLabel">
       <property name="text">
        <string>Pr√©nom</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="saveParameters">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="readCard">
     <property name="text">
      <string>Lire carte</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
